redis/client.py

```diff
@@ -1009,13 +1021,27 @@ def acl_setuser(self, username, enabled=False, nopass=False,
             for password in remove_passwords:
                 pieces.append('<%s' % password)
 
+        if remove_hashes:
+            # as most users will have only one password, allow remove_passwords
+            # to be specified as a simple string or a list
+            remove_hashes = list_or_args(remove_hashes, [])
+            for password in remove_hashes:
+                pieces.append('!%s' % password)
+
         if add_passwords:
             # as most users will have only one password, allow add_passwords
             # to be specified as a simple string or a list
             add_passwords = list_or_args(add_passwords, [])
             for password in add_passwords:
                 pieces.append('>%s' % password)
 
+        if add_hashes:
+            # as most users will have only one password, allow remove_passwords
+            # to be specified as a simple string or a list
+            add_hashes = list_or_args(add_hashes, [])
+            for password in remove_hashes:
```

Comment:
This should be `for password in add_hashes:`

Reply:
Just pushed this one.